src/backend/electron/main.ts

```diff
@@ -584,7 +584,12 @@ registerIpcMainHandle<IpcMainHandle>({
       dialog.showSaveDialog(win, {
         title,
         defaultPath,
-        filters: [{ name: "Wave File", extensions: ["wav"] }],
+        filters: [
+          {
+            name: "wavファイル",
```

Comment:
（細かいけど）
なんか公式はWAVファイルっぽい！
```suggestion
            name: "WAVファイル",
```